das_argument_uint64 does not exist even though its declared in daScript.h

daScript.h has a declaration for `das_argument_uint64` even though only `das_argument_uint64_unaligned` exists


Discovered when building our project:
```
/usr/bin/ld: /home/relt/.dub/cache/pukeko/~main/build/application-debug-TAjsqRWK9B_FEbdLNAOK-w/pukeko.o: in function `D_ComponentFuncAfter':
/media/relt/M2/AutismGame/Engine/source/script.d:198:(.text.D_ComponentFuncAfter[D_ComponentFuncAfter]+0x48): undefined reference to `das_argument_uint64'
collect2: error: ld returned 1 exit status
Error: undefined reference to `das_argument_uint64`
       referenced from `D_ComponentFuncAfter`
       perhaps a library needs to be added with the `-L` flag or `pragma(lib, ...)`
Error: linker exited with status 1
       cc /home/relt/.dub/cache/pukeko/~main/build/application-debug-TAjsqRWK9B_FEbdLNAOK-w/pukeko.o -o /home/relt/.dub/cache/pukeko/~main/build/application-debug-TAjsqRWK9B_FEbdLNAOK-w/pukeko -g -m64 -Xlinker --no-as-needed -Xlinker --export-dynamic /home/relt/.dub/cache/bindbc-assimp5/0.1.0/build/library-debug-pxG0fBC99_G_W0MNW90WAw/libbindbc-assimp5.a /home/relt/.dub/cache/bindbc-glfw/1.1.2/build/dynamic-debug-8KkoHpSBDfyEvk6bwOf9rQ/libBindBC_GLFW.a /home/relt/.dub/cache/bindbc-opengl/1.1.1/build/dynamic-debug-9yiOdMGzI6MV4X-MvYB5JQ/libBindBC_OpenGL.a /home/relt/.dub/cache/bindbc-loader/1.1.5/build/noBC-debug--gBnV1dcTnithk8A_Mefpg/libBindBC_Loader.a /home/relt/.dub/cache/sdlang-d/0.10.6/build/library-debug-I1TSBguaIcUl9cAa4DLVhw/libsdlang-d.a /home/relt/.dub/cache/taggedalgebraic/0.11.24/build/library-debug-moD6jM-lmFHs24YIZkh0OQ/libtaggedalgebraic.a -Lextern/daScript/lib -Lextern/monos/build -lglfw -llibDaScript -llibUriParser -lstdc++ -lm -ldl -lpthread -lmonoswrp -L/usr/lib/x86_64-linux-gnu -Xlinker -Bstatic -lphobos2 -Xlinker -Bdynamic -lpthread -lm -lrt -ldl 
Error /usr/bin/dmd failed with exit code 1.
```

--- a/src/misc/daScriptC.cpp
+++ b/src/misc/daScriptC.cpp
@@ -510,6 +510,8 @@ char * das_allocate_string ( das_context * context, char * str ) {
 
 int    das_argument_int ( vec4f arg ) { return cast<int>::to(arg); }
 unsigned int   das_argument_uint ( vec4f arg ) { return cast<unsigned int>::to(arg); }
+long long das_argument_int64 ( vec4f arg ) { return cast<long long>::to(arg); }
+unsigned long long das_argument_uint64 ( vec4f arg ) { return cast<unsigned long long>::to(arg); }
 int    das_argument_bool ( vec4f arg ) { return cast<bool>::to(arg) ? 1 : 0; }
 float  das_argument_float ( vec4f arg ) { return cast<float>::to(arg); }
 double  das_argument_double ( vec4f arg ) { return cast<double>::to(arg); }

